invenio/modules/search/enhancers/basic_enhancers.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Very simple enhancer functions used as proof of concept
+
+    Every enhancer function accepts the SearchEngine instance as the first
+    parameter and an arbitrary number of parameters and keyword values defined
+    in the decorator. It should return the new enhanced query AST.
+"""
+
+
+def identical_enhancer(search_obj, *args, **kwargs):
+    return search_obj.query
```

Comment:
if it's "identity" on first argument then we should return `search_obj`.

--
PS: `s/search_obj/query/`